backend/module/eCampCore/src/Hydrator/ContentTypeConfigHydrator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace eCamp\Core\Hydrator;
+
+use eCamp\Core\Entity\ContentTypeConfig;
+use eCamp\Lib\Hydrator\Util;
+use Laminas\Hydrator\HydratorInterface;
+
+class ContentTypeConfigHydrator implements HydratorInterface {
+    public static function HydrateInfo() {
+        return [
+            'contentType' => Util::Entity(function (ContentTypeConfig $e) {
+                return $e->getContentType();
+            }),
+        ];
+    }
+
+    /**
+     * @param object $object
+     *
+     * @return array
+     */
+    public function extract($object) {
+        /** @var ContentTypeConfig $contentTypeConfig */
+        $contentTypeConfig = $object;
+
+        return [
+            'id' => $contentTypeConfig->getId(),
+            'required' => $contentTypeConfig->getRequired(),
+            'multiple' => $contentTypeConfig->getMultiple(),
+        ];
+    }
+
+    /**
+     * @param object $object
+     *
+     * @return object
+     */
+    public function hydrate(array $data, $object) {
+        /** @var ContentTypeConfig $contentTypeConfig */
+        $contentTypeConfig = $object;
+
+        if (isset($data['required'])) {
+            $contentTypeConfig->setRequired($data['required']);
+        }
+        if (isset($data['multiple'])) {
+            $contentTypeConfig->setMultiple($data['multiple']);
+        }
+
```

Comment:
`defaultInstances` gibts in diesem PR noch nicht, ich nehme an das kommt im nächsten dann rein?

Reply:
Ja - oder sogarn noch ein bissel später (nach einer Diskussion).

Ich bin der Meinung, dass eine Zahl alleine wenig hilft. Ich weiss nicht, was ich mit zwei "StoryBoard" ActivityContents soll, wenn diese nicht mit deren Absicht beschriftet sind. z.B. "Schönweter" und "Schlechtwertter".

Daher möchte ich hinterfrage, ob hier eine Zahl alleine ausreicht - oder ob wir nicht mehr Information benötigen.
Und falls mehr Information - was alles?